### ISSUE TITLE
Using ../ to go to folder above

### ISSUE THREAD

**Issue by Drullkus** [Chisel-Team/Chisel#269]
As requested by @BluSunrize,

Currently, we use `./` as a shortcut to navigate to the folder the texture is in. Blu is suggesting that we have `../` to go to folder above. This actually makes sense in terms of organization, as two known mods so far adopting CTM have made dedicated CTM folders to store extra files related to CTM and related assets.

**Comment by BluSunrize** [comment]
It's not an urgent thing, I just figured it's a nice comfort feature for those people that want to implement CTM compat within their mods have have, like me, chosen to put all the textures in a subfolder (textures/blocks/ctm).
Having a path that allows me to just go up one level would make the file look neater =P

**Comment by tterrag1098** [comment]
Yeah, currently the directory expansion is...lazy. I need to make it proper :P
